lib/contractWrapperBase.ts

```diff
@@ -192,28 +192,53 @@ export abstract class ContractWrapperBase {
     ): EventFetcher<TArgs> => {
 
       let baseEvent: EventFetcher<TArgs>;
+      let receivedEvents: Set<Hash>;
+
+      if (!!filterObject.suppressDups) {
+        receivedEvents = new Set<Hash>();
+      }
+
+      const handleEvent = (
+        error: any,
+        log: DecodedLogEntryEvent<TArgs> | Array<DecodedLogEntryEvent<TArgs>>,
+        callback?: EventCallback<TArgs>): void => {
+
+        /**
+         * always provide an array
```

Comment:
question: do these small comments appear in the generated docs? (because of the double asteriks)

Reply:
> do these small comments appear in the generated docs? (because of the double asteriks)

Not when they appear inside a function like that